src/EnergyPlus/IceThermalStorage.cc

```diff
@@ -886,517 +700,442 @@ namespace IceThermalStorage {
+                                                                                           DataLoopNode::NodeConnectionType_Inlet,
+                                                                                           1,
+                                                                                           DataLoopNode::ObjectIsNotParent);
 
             // Get Plant Outlet Node Num
-            IceStorage(IceNum).PltOutletNodeNum = GetOnlySingleNode(
-                cAlphaArgs(4), ErrorsFound, cCurrentModuleObject, cAlphaArgs(1), NodeType_Water, NodeConnectionType_Outlet, 1, ObjectIsNotParent);
+            SimpleIceStorage(iceNum).PltOutletNodeNum = NodeInputManager::GetOnlySingleNode(DataIPShortCuts::cAlphaArgs(4),
+                                                                                            ErrorsFound,
+                                                                                            DataIPShortCuts::cCurrentModuleObject,
+                                                                                            DataIPShortCuts::cAlphaArgs(1),
+                                                                                            DataLoopNode::NodeType_Water,
+                                                                                            DataLoopNode::NodeConnectionType_Outlet,
+                                                                                            1,
+                                                                                            DataLoopNode::ObjectIsNotParent);
 
             // Test InletNode and OutletNode
-            TestCompSet(cCurrentModuleObject, cAlphaArgs(1), cAlphaArgs(3), cAlphaArgs(4), "Chilled Water Nodes");
+            BranchNodeConnections::TestCompSet(DataIPShortCuts::cCurrentModuleObject,
+                                               DataIPShortCuts::cAlphaArgs(1),
+                                               DataIPShortCuts::cAlphaArgs(3),
+                                               DataIPShortCuts::cAlphaArgs(4),
+                                               "Chilled Water Nodes");
 
             // Initialize Report Variables
-            IceStorageReport(IceNum).MyLoad = 0.0;
-            IceStorageReport(IceNum).U = 0.0;
-            IceStorageReport(IceNum).Urate = 0.0;
-            IceStorageReport(IceNum).IceFracRemain = 1.0;
-            IceStorageReport(IceNum).ITSCoolingRate = 0.0;
-            IceStorageReport(IceNum).ITSCoolingEnergy = 0.0;
-            IceStorageReport(IceNum).ITSChargingRate = 0.0;
-            IceStorageReport(IceNum).ITSChargingEnergy = 0.0;
-            IceStorageReport(IceNum).ITSmdot = 0.0;
-            IceStorageReport(IceNum).ITSInletTemp = 0.0;
-            IceStorageReport(IceNum).ITSOutletTemp = 0.0;
+            SimpleIceStorage(iceNum).MyLoad = 0.0;
+            SimpleIceStorage(iceNum).Urate = 0.0;
```

Comment:
I see the `U` member variable is no longer initialized to zero here.  I'm sure there is a reason, just noting it.